GitUp/Application/AppDelegate.m

```diff
@@ -613,6 +620,10 @@ - (IBAction)cloneRepository:(id)sender {
   }
 }
 
+- (IBAction)cloneRepository:(id)sender {
+  [self _cloneRepositoryFromURLString:@""];
```

Comment:
That's fine. I haven't worked on this UI in forever, so I don't remember all the details 😉